kythe/cxx/indexer/cxx/KytheGraphObserver.h

```diff
@@ -159,8 +159,8 @@ class KytheGraphObserver : public GraphObserver {
     return &default_token_;
   }
 
-  void applyMetadataFile(clang::FileID ID, const clang::FileEntry* FE,
-                         const std::string& SearchString) override;
```

Comment:
Basically all of the case changes in this file should either be omitted or applied everywhere (rather than just the ones found by clang-tidy).